graphene_django/filter/tests/test_fields.py

```diff
@@ -321,12 +321,14 @@ class Query(ObjectType):
         pub_date=datetime.now(),
         pub_date_time=datetime.now(),
         reporter=r1,
+        editor=r1,
```

Comment:
This seems out of place, but I am also not sure how Articles were being created with a null editor....

Reply:
@zbyte64 flake8 raised the fact that this test name was the same as another one further down the file and so it was never being run. So when I renamed that test (https://github.com/graphql-python/graphene-django/pull/688/files#diff-42247536fb4398767212e74574263a3bR455) this one started failing because of the missing `editor` param.